doc/source/how-to/releasing.rst

```diff
@@ -212,39 +212,139 @@ script. If you are operating out of a GitHub CI pipeline, email the PyAnsys Core
 team at `pyansys.core@ansys.com <mailto:pyansys.core@ansys.com>`_ for the
 required ``PAT`` user name and ``PYANSYS_PYPI_PRIVATE_PAT`` password.
 
-Assuming that you are already in a Python repository's ``dist/`` directory that
-contains your wheel, source distribution, or both, you can upload to the private
-repository with:
+For uploading packages, `Twine <https://pypi.org/project/twine/>`_ is required.
+Install this tool by running:
 
-.. code-block:: bash
+.. code-block:: shell
 
-   pip install -U pip build twine
+    python -m pip install twine
 
-   REPOSITORY_URL="https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload"
-   python -m twine upload dist/* \
-     -p ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }} \
-     -u PAT \
-     --repository-url $REPOSITORY_URL
 
+.. note:: 
+
+   Once a library has been made public, there is no need to keep uploading new
+   versions of it to the private PyPI.
+
+Uploading to private PyPI using GitHub actions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The following code allows to publish any Python :ref:`Artifacts` contained in
+the ``dist/`` directory to the private PyPI. It is expected to be included when
+:ref:`Using GitHub actions`:
+
+.. code-block:: yaml
+
+    - name: Publish to private PyPI
+      env:
+        TWINE_USERNAME: PAT
+        TWINE_PASSWORD: ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }}
+        TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload
+      run: |
+        pip install twine
+        python -m twine upload dist/* \
+
+
+Notice that ``PYANSYS_PYPI_PRIVATE_PAT`` needs to be added as a repository
+secret so its value is available during the execution of previous instructions.
+
+Uploading to private PyPI using the command-line
```

Comment:
```suggestion
Uploading to private PyPI using the command line
```

Reply:
No hyphen except when command-line is a a compound adjective, such as in command-line tool arguments in the next line.